tests/webapps/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Google Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.runtimes.jetty.tests</groupId>
+    <artifactId>tests-parent</artifactId>
+   <version>0.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>tests-webapps-parent</artifactId>
+  <name>Jetty-Runtime :: Tests :: WebApps Parent</name>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>test-war-hello</module>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.eclipse.jetty.toolchain</groupId>
+        <artifactId>jetty-test-helper</artifactId>
+        <version>4.0</version>
+        <scope>test</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/docker</directory>
+        <filtering>true</filtering>
+        <targetPath>${project.build.directory}/docker</targetPath>
+      </resource>
+    </resources>
+    <plugins>
+      <plugin>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <configuration>
+          <!-- 
+             Tests cannot and should not be validated with checkstyle.
+             these tests need to be flexible enough for good, bad, quirky
+             and everything in between with regards to compliance and
+             validity.
+             Having them pass checkstyle makes them invalid for integration
+             test cases
+             -->
+          <skip>true</skip>
```

Comment:
This is not advisable for the webapp's own code. (if you want to replicate issues from end users you need to be able to produce code that isn't "clean" or "accurate" to ensure that you are producing accurate regression testing)
I'm fine with enabling checkstyle for `src/test/java` if you want to do it for the test cases themselves, but not the implementing code.